src/main/java/it/pagopa/transactions/commands/handlers/TransactionSendClosureHandler.java

```diff
@@ -141,37 +138,31 @@ public Mono<Either<TransactionClosureErrorEvent, TransactionEvent<TransactionClo
-                                            "authorizationCode",
-                                            authRequestData.authorizationCode(),
-                                            "rrn",
-                                            ECOMMERCE_RRN,
-                                            "fee",
-                                            fee.toString(),
-                                            "timestampOperation",
-                                            // FIXME Pass the timestamp of the authorization result
-                                            // bug CHK-1410: date formatted to yyyy-MM-ddTHH:mm:ss truncating millis
-                                            OffsetDateTime.now()
-                                                    .toLocalDateTime()
-                                                    .truncatedTo(ChronoUnit.SECONDS)
-                                                    .format(DateTimeFormatter.ISO_LOCAL_DATE_TIME),
-                                            "totalAmount",
-                                            totalAmount.toString()
-                                    )
-                            );
-
+                            .transactionId(tx.getTransactionId().value());
+
+                    if (ClosePaymentRequestV2Dto.OutcomeEnum.OK.equals(closePaymentRequest.getOutcome())) {
+                        closePaymentRequest.idPSP(transactionAuthorizationRequestData.getPspId())
+                                .idBrokerPSP(transactionAuthorizationRequestData.getBrokerName())
+                                .idChannel(transactionAuthorizationRequestData.getPspChannelCode())
+                                .totalAmount(totalAmount)
+                                .fee(fee)
+                                .timestampOperation(updateAuthorizationRequestDto.getTimestampOperation())
+                                .paymentMethod(transactionAuthorizationRequestData.getPaymentTypeCode())
+                                .additionalPaymentInformations(
+                                        new AdditionalPaymentInformationsDto()
+                                                .tipoVersamento(TIPO_VERSAMENTO_CP)
+                                                .outcomePaymentGateway(
+                                                        AdditionalPaymentInformationsDto.OutcomePaymentGatewayEnum
+                                                                .fromValue(authRequestData.outcome())
+                                                )
+                                                .authorizationCode(authRequestData.authorizationCode())
+                                                .fee(fee)
+                                                .timestampOperation(
+                                                        updateAuthorizationRequestDto.getTimestampOperation()
+                                                )
```

Comment:
the format must be yyyy-MM-ddTHH:mm:ss

Reply:
See https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/197/files#diff-6b0cfdb8c858704496a399b24100164135349bfeac8605860fb1a7c7e6134d98R159-R164